### ISSUE TITLE
Make custom commands more visible in the GUI

### ISSUE THREAD

**Issue by chbiel** [cypress-io/cypress#6645]
### Current behavior:

Currently, when using custom commands defined with `Cypress.Commands.add('someCommand', () => {});` they are not clearly visible in the GUI what makes debugging hard when you have a more complex test with several of those commands.

### Desired behavior:

Group commands in the GUI to make visible that a command is issued through a custom command.
Groups are only one option. Possible alternatives could be alternating colors of the command rows. Or indent the commands or wrap them in another collapsible.

As this is a GUI enhancement, issues like https://github.com/cypress-io/cypress/issues/4858 are related and maybe makes this obsolet when it's (e.g. in this case) clear, that the problem came from a command.

**Comment by jennifer-shehane** [comment]
You're aware of `Cypress.log()`? This can be used to control the logging of custom commands to the Command Log. https://on.cypress.io/cypress-log

Additionally you can turn off all the other logging of commands within the custom commands if it's really noisy like:

```js
Cypress.Commands.add('login', (username, pw) => {
  cy.visit('index.html', { log: false }) // turn off logging of this command

  Cypress.log({
    name: 'login',
    message: `${username}, ${pw}`
  })
})

it('url', () => {
  cy.login('jen@email.com', 'abc123')
})
```

<img width="498" alt="Screen Shot 2020-03-05 at 3 20 51 PM" src="https://user-images.githubusercontent.com/1271364/75964034-eb3e9600-5ef4-11ea-898f-44448d9425a0.png">

The docs are...pretty bad, so maybe I will update them. Does this meet the requirements you're asking for?

**Comment by chbiel** [comment]
Thanks! Actually I was not aware of `Cypress.log()`, maybe, as you said, that the documentation not clearly says what the function does :)
Would using your example mean that, on error, I get the error for that log or do failed assertions etc. still output errors as normal (although `log: false` is set)?

I will try it out if this would solve my problem of having a better debugging experience.

Additionally: is there a way to overwrite `Cypress.Commands.add` to attach a log to all issued commands?

**Comment by jennifer-shehane** [comment]
@chbiel I've updated the example in the docs so hopefully it is clearer - it is adding logs to a `Cypress.Command.add` custom command. https://on.cypress.io/cypress-log#Examples

I don't think this should have any effect on errors displaying in the Command Log, but I'd have to try it out.

>Additionally: is there a way to overwrite Cypress.Commands.add to attach a log to all issued commands?

I'm not totally sure what you mean by this. This is what `Cypress.log` does essentially if it's written within `Cypress.Command.add`.

Closing as resolved since custom logging is already possible with custom commands.
